src/frontend/src/lib/types/token.ts

```diff
@@ -40,6 +40,7 @@ export type OptionTokenStandard = TokenStandard | undefined | null;
 export type TokenToPin = Pick<Token, 'id'> & { network: Pick<Token['network'], 'id'> };
 
 interface TokenFinancialData {
+	balance?: number;
```

Comment:
I would keep non formatted values in stores, in that case `BigNumber` which feels more future prone. If it's more handy to hold a formatted value in the store then I would name the variable accordingly, like `formattedBalance` or something.

Reply:
make sense! however is more useful to have the formatted one: i will rename it, but if you prefer i can add even the `BigNumber` prop, having two new values: `balance` and `formattedBalance`

Reply:
Sounds good